README.md

```diff
@@ -17,27 +17,61 @@ Integration branch status: [![Integration Branch Build Status](https://travis-ci
 
 This repository acts as the source code location for [the Unicorn Utterances blog](https://unicorn-utterances.com)
 
-## Important Files
+## Sponsors
+
+[![The Polyglot Developer](./static/sponsors/the-polyglot-developer.svg)](https://www.thepolyglotdeveloper.com/)
+
+## Statement of Ethics
+
+We never want to end up in a place where our educational content, the
+experience around that content, or our community is compromised; neither by
+financial sway or by any potentially harmful members of the community. As a
+result, [we've implemented the Contributor Covenant as our code of conduct](CODE_OF_CONDUCT.md).
```

Comment:
Suggested change:

```diff
- We never want to end up in a place where our educational content, the
- experience around that content, or our community is compromised; neither by
- financial sway or by any potentially harmful members of the community. As a
- result, [we've implemented the Contributor Covenant as our code of conduct](CODE_OF_CONDUCT.md).
+ We never want to end up in a place where our educational content, experience,
+ or community is compromised by either financial sway or potentially harmful
+ members of the community. As such, we've implemented the
+ [Contributor Covenant](https://www.contributor-covenant.org/)
+ as our [code of conduct](CODE_OF_CONDUCT.md) to uphold these values.
```

Says basically the same thing, just more concise and a bit more readable (IMO) without a semicolon in the middle.